### PR TITLE
Vercel image optimisation support

### PR DESCRIPTION
**Pull request by istarkov**
## Description

CLI support of vercel image optimisation

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: packages/cli/templates/defaults/app/constants.ts [hunk 1]
```diff
@@ -1,2 +1,24 @@
...
 export const imageBaseUrl = "/assets/";
+
+export const imageLoader: ImageLoader = ({ quality, src, width }) => {
+  if (process.env.NODE_ENV !== "production") {
+    return imageBaseUrl + src;
+  }
+
+  const searchParams = new URLSearchParams();
+  searchParams.set("w", width.toString());
+  searchParams.set("q", quality.toString());
```
Comment (JayaKrishnaNamburu) [inline packages/cli/templates/defaults/app/constants.ts:13]: Looks like the `searchParams` is not being used 🤔

Comment (istarkov) [reply packages/cli/templates/defaults/app/constants.ts:13]: Thanks

Comment (istarkov) [reply packages/cli/templates/defaults/app/constants.ts:13]: Strange that checks completed

Comment (JayaKrishnaNamburu) [reply packages/cli/templates/defaults/app/constants.ts:13]: Maybe they are considering the `searchParams.set` as declared and used !!

**Review by TrySound — APPROVED** [review]

**Pull request merged**